common/models/model-definition.json

```diff
@@ -96,7 +96,8 @@
     "methods": {
       "embed": {
         "as": "object",
-        "key": "name"
+        "key": "name",
```

Comment:
`key` needs to be there so it gets removed before writing to JSON.
https://github.com/strongloop/loopback-workspace/pull/273/files#diff-36dfec9040d3e62a2c6ca85e70ab0e7bL176